tests/resources/test_dashboard.py

```diff
@@ -29,3 +52,364 @@ def test_dashboard(self):
         dashboard = self.jira.dashboard(expected_ds.id)
         self.assertEqual(dashboard.id, expected_ds.id)
         self.assertEqual(dashboard.name, expected_ds.name)
+
+    @pytest.mark.skipif(
+        os.environ.get("CI_JIRA_TYPE", "Server").upper() != "CLOUD",
+        reason="Functionality only available on Jira Cloud",
+    )
```

Comment:
Let's make this into a similar style as ``@allow_on_cloud`` maybe the name can be only_run_on_cloud